packages/pn-pa-webapp/src/components/Statistics/EmptyStatistics.tsx

```diff
@@ -1,18 +1,59 @@
 /* eslint-disable functional/immutable-data */
 import React from 'react';
-import { useTranslation } from 'react-i18next';
+import { Trans, useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
 
-import { Box, Stack, SxProps, Theme, Typography } from '@mui/material';
+import { Box, Link, Stack, SxProps, Theme, Typography } from '@mui/material';
 import { IllusStatistics } from '@pagopa-pn/pn-commons';
 
+import * as routes from '../../navigation/routes.const';
+
 type Props = {
   description?: string;
   sx?: SxProps<Theme>;
 };
 
+type LinkDashboardProps = {
+  children?: React.ReactNode;
+};
+
+const LinkDashboard: React.FC<LinkDashboardProps> = ({ children }) => {
+  const { t } = useTranslation(['statistics']);
+  const navigate = useNavigate();
+
+  return (
+    <Link
```

Comment:
The style of the link doesn't is the the same of the figma. Maybe we have to use the ButtonNaked component. Check other links around the application